content/en/docs/setup/additional-setup/sidecar-injection/index.md

```diff
@@ -134,141 +134,6 @@ sleep-776b7bcdcd-bhn9m   2/2       Terminating   0          2m
-      app: ignored
-  template:
-    metadata:
-      labels:
-        app: ignored
-      annotations:
-        sidecar.istio.io/inject: "false"
-    spec:
-      containers:
-      - name: ignored
-        image: governmentpaas/curl-ssl
-        command: ["/bin/sleep","infinity"]
-EOF
-{{< /text >}}
-
-##### _**template**_
-
-The sidecar injection template uses <https://golang.org/pkg/text/template> which,
-when parsed and executed, is decoded to the following
-struct containing the list of containers and volumes to inject into the pod.
-
-{{< text go >}}
-type SidecarInjectionSpec struct {
-      RewriteAppHTTPProbe bool                          `yaml:"rewriteAppHTTPProbe"`
-      InitContainers      []corev1.Container            `yaml:"initContainers"`
-      Containers          []corev1.Container            `yaml:"containers"`
-      Volumes             []corev1.Volume               `yaml:"volumes"`
-      DNSConfig           *corev1.PodDNSConfig          `yaml:"dnsConfig"`
-      ImagePullSecrets    []corev1.LocalObjectReference `yaml:"imagePullSecrets"`
-}
-{{< /text >}}
-
-The template is applied to the following data structure at runtime.
-
-{{< text go >}}
-type SidecarTemplateData struct {
-    DeploymentMeta *metav1.ObjectMeta
-    ObjectMeta     *metav1.ObjectMeta
-    Spec           *corev1.PodSpec
-    ProxyConfig    *meshconfig.ProxyConfig  // Defined by https://istio.io/docs/reference/config/service-mesh.html#proxyconfig
-    MeshConfig     *meshconfig.MeshConfig   // Defined by https://istio.io/docs/reference/config/service-mesh.html#meshconfig
-}
-{{< /text >}}
-
-`ObjectMeta` and `Spec` are from the pod. `ProxyConfig` and `MeshConfig`
-are from the `istio` ConfigMap in the `istio-system` namespace. Templates can conditionally
-define injected containers and volumes with this data.
-
-For example, the following template
-
-{{< text plain >}}
-containers:
-- name: istio-proxy
-  image: istio.io/proxy:0.5.0
-  args:
-  - proxy
-  - sidecar
-  - --configPath
-  - {{ .ProxyConfig.ConfigPath }}
-  - --binaryPath
-  - {{ .ProxyConfig.BinaryPath }}
-  - --serviceCluster
-  {{ if ne "" (index .ObjectMeta.Labels "app") -}}
-  - {{ index .ObjectMeta.Labels "app" }}
-  {{ else -}}
-  - "istio-proxy"
-  {{ end -}}
-{{< /text >}}
-
-expands to
-
-{{< text yaml >}}
-containers:
-- name: istio-proxy
-  image: istio.io/proxy:0.5.0
-  args:
-  - proxy
-  - sidecar
-  - --configPath
-  - /etc/istio/proxy
-  - --binaryPath
-  - /usr/local/bin/envoy
-  - --serviceCluster
-  - sleep
-{{< /text >}}
-
-when applied over a pod defined by the pod template spec in [`samples/sleep/sleep.yaml`]({{< github_tree >}}/samples/sleep/sleep.yaml)
-
 #### More control: adding exceptions
 
 There are cases where users do not have control of the pod creation, for instance, when they are created by someone else. Therefore they are unable to add the annotation `sidecar.istio.io/inject` in the pod, to explicitly instruct Istio whether to install the sidecar or not.
```

Comment:
```
the annotation `sidecar.istio.io/inject` in the pod
```
This is referring to an annotation that doesn't seem to be documented, if you remove all the previous material. Is this still implemented, and if so, where is it documented now?